### ISSUE TITLE
Error: Cannot find module 'babel-plugin-module-resolver'. Maybe babel7 support issue.

### ISSUE THREAD

**Issue by hyochan** [necolas/react-native-web#1202]
**Is your feature request related to a problem? Please describe.**
Today, I am trying to embed `react-native-web` into my `boilerplate` which belongs to `https://github.com/dooboolab/dooboo-native-js/tree/web-support` repo. I've been working on this whole day, and currently stuck with below issue. It is working fine before I met `decorator` syntax.
```
ERROR in ./index.web.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'babel-plugin-module-resolver' from '/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js'
    at Function.module.exports [as sync] (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/resolve/lib/sync.js:43:15)
    at resolveStandardizedName (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at items.map (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at plugins (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-descriptors.js:40:19)
    at mergeChainOpts (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-chain.js:319:26)
    at /Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-chain.js:283:7
    at buildRootChain (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/config-chain.js:120:22)
    at loadPrivatePartialConfig (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/partial.js:85:55)
    at Object.loadPartialConfig (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/@babel/core/lib/config/partial.js:110:18)
    at Object.<anonymous> (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:140:26)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:3:103)
    at _next (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:5:194)
    at /Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:5:364
    at new Promise (<anonymous>)
    at Object.<anonymous> (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:5:97)
    at Object._loader (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:220:18)
    at Object.loader (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:56:18)
    at Object.<anonymous> (/Users/hyochan/SourceCode/DoobooProjects/dooboo-native-js/node_modules/babel-loader/lib/index.js:51:12)
 @ multi @babel/polyfill ./index.web.js main[1]
```

**Describe a solution you'd like**
If you go to above [repository](https://github.com/dooboolab/dooboo-native-js/tree/web-support) I've linked, you can find my `babel` and `webpack` setup which I am not using `babel-plugin-module-resolver` and instead I am using `@babel/plugin-proposal-decorators`.

**Describe alternatives you've considered**
I feel like this is lack of support of `babel7` currently, if I understand correctly.

**Additional context**
Could you see my config in [repository](https://github.com/dooboolab/dooboo-native-js/tree/web-support) and give me any suggestion? In [index.web.js](https://github.com/dooboolab/dooboo-native-js/blob/web-support/index.web.js), if I remove `import appStore from './src/stores/appStore';` in line 8, my `webpack` compiles correctly which doesn't include `babel decorator` even my project is configured as `babel 7`.

Thanks. I really need any suggestion to move forward. Currently I am stuck.

**Comment by necolas** [comment]
This is a problem in your project setup and has nothing to do with his this project is internally built. Please refer to the Babel docs and make sure all your plugins are compatible with the version of Babel you are using

**Comment by hyochan** [comment]
@necolas If you tell me that I will look into this further. However, all my projects setup with `babel7` has been working quite well for a long time there are several boilerplates I've already maintained list up [here](https://github.com/orgs/dooboolab/teams/react-native-boilerplates/repositories). I hope someone to try `react-native-web` with `babel7` and the `mobx` and check that the `decorator syntax` is working correctly. Because it is keep seeking for `babel-plugin-module-resolver` which isn't `babel7` compatible even though I am not using it in my projects. It just throwing that error after I've imported `react-native-web`.
Hope you've understood my symptoms correctly and thanks for creating this.
